src/spec/openapi.ts

```diff
@@ -112,6 +112,31 @@ export interface paths {
 			}
 		}
 	}
+	'/policies/{policyId}/token': {
+		post: {
```

Comment:
```suggestion
		get: {
```

Reply:
yes ser

Reply:
Just need to run `yarn codegen` to update this.